src/XrmContext/Interpretation/InterpretEntityMetadata.fs

```diff
@@ -243,7 +243,7 @@ let interpretEntity entityNames entityMap entityToIntersects deprecatedPrefix la
   let desc = getDescription (getLabelOption metadata.DisplayName) metadata.Description
 
   // Return the entity representation
-  { XrmEntity.typecode = metadata.ObjectTypeCode.GetValueOrDefault()
+  { XrmEntity.typecode = if includeEntityTypeCode then Some(metadata.ObjectTypeCode.GetValueOrDefault()) else None
```

Comment:
The XrmEntity. prefix is not needed here. I know it was there before, but we might as well clean up :).

It is also prettier to pipe arguments instead of wrapping in parenthese.

Thus, the line should just look like `typecode = if includeEntityTypeCode then metadata.ObjectTypeCode.GetValueOrDefault() |> Some else None`

Reply:
Thanks for the feedback and sorry for the delay in applying it, Damon did the work I just forgot to pull it through!